### PR TITLE
1107 - TYDE2 - TYDE nav bar should show text for each tab on hover

### PR DESCRIPTION
**Pull request by EldadZZipori**
## Changes
- adding text to the tyde navbar on hover
## Screenshot

https://user-images.githubusercontent.com/50620330/129635541-225e257a-ceaa-4dc9-9640-efc8de29f05c.mov

## Issue Linkage
Closes (issue #1107 )
## PR Dependency
Depends on: (N/A)
## Automated Testing
Tyde, no automated testing

### CODE REVIEW

**Comment by Frozemint** [comment]
Is the base branch for this PR correct? I am seeing a million changes in the files changed section

File: templates/vue/src/components/tyde/Navbar.vue [hunk 1]
```diff
@@ -3,15 +3,19 @@
...
           :key="tab"
           :ref="tab + '-tab'"
           active-class="active"
           link-classes="link"
           :active="selectedTab === tab"
           @click="handleTabChange(tab)"
         >
-          <tyde-icon :selected="isSelectedTap(tab)" :icon="tab" />
+          <tyde-icon
+            :selected="isSelectedTap(tab)"
```
Comment (odotadot) [inline templates/vue/src/components/tyde/Navbar.vue:15]: `isSelectedTab`?

File: templates/vue/src/components/tyde/TydeIcon.vue [hunk 2]
```diff
@@ -31,19 +40,49 @@ export default {
...
+          title = "Circle of Support"
+          break
+      }
+
+      return title
+    },
+  },
   watch: {
     selected() {
       this.selectIcon()
```
Comment (odotadot) [inline templates/vue/src/components/tyde/TydeIcon.vue:77]: In the `selectIcon()` method, can simplify each case to a ternary expression like `selectedIcon = this.selected ? tyde : tydeUnselected`

**Comment by odotadot** [comment]
Tested nav bar text and the functionality works great. A bit strange that the Circle of Support text shows on the left while all the other navigation text is on the right, but it looks like there were some limitations that caused this? Code looks clean.

File: templates/vue/src/components/tyde/Navbar.vue (hunk 1 above)
Comment (EldadZZipori) [reply templates/vue/src/components/tyde/Navbar.vue:15]: Yep.

File: templates/vue/src/components/tyde/TydeIcon.vue (hunk 2 above)
Comment (EldadZZipori) [reply templates/vue/src/components/tyde/TydeIcon.vue:77]: Nice catch

**Comment by EldadZZipori** [comment]
Note that COS text shows on the left in the mock-up as well. There is also no space for it on the right.

**Review by odotadot — APPROVED** [review]

**Review by wynnset — APPROVED** [review]

**Pull request merged**